proteus/Domain.py

```diff
@@ -17,6 +19,17 @@ def __init__(self,nd,name="defaultDomain",units="m"):
         self.units=units
         self.x=[]#minx,miny,minz
         self.L=[]#bounding box when self.x is origin
+        self.g=g
```

Comment:
@cekees I attached it there but actually don't actually use it from there yet. 
As gravity is only defined once, I thought that it would be convenient if we need to access it somewhere else, such as predefined boundary conditions. This could be moved somewhere else ultimately.